extensions/SemanticResultFormats/formats/Exhibit/exhibit/exhibit-api.js

```diff
@@ -169,7 +169,7 @@
         }
 
         if (useLocalResources) {
-            Exhibit.urlPrefix = wgServer + wgScriptPath + "/extensions/SemanticResultFormats/formats/Exhibit/exhibit/";
+            Exhibit.urlPrefix = wgScriptPath + "/extensions/SemanticResultFormats/formats/Exhibit/exhibit/";
         }
 
         if (Exhibit.params.locale) { // ISO-639 language codes,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)

[project-id]: wikia:app